packages/amuse-aarsethzare/setup.py

```diff
@@ -15,7 +13,7 @@
 license_ = "Apache License 2.0"
 url = 'http://www.amusecode.org/'
 install_requires = [
-    'amuse-framework>=%s' % (main_version),
+    'amuse-framework',
```

Comment:
why does the version requirement dissappear?

Reply:
Mainly because "main version" is no longer valid with date-based updates. 
We needed this to prevent incompatible versions being installed together.
Maybe we now need to hardcode this instead.

Reply:
(and update this whenever an incompatible update is pushed to amuse-framework)